docs/gen-ai/gen-ai-evaluation-events.md

```diff
@@ -0,0 +1,50 @@
+
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Generative AI evaluation events
+--->
+
+# Semantic Conventions for GenAI evaluation events
+
+**Status**: [Experimental][DocumentStatus]
+
+Each evaluation event defines a common way to report an evaluation score and the context for this specific evaluation method.
+
+## Naming pattern
+
+The evaluation events follow `gen_ai.evaluation.{evaluation method}` naming pattern.
+For example, evaluations that are common across different GenAI models and framework tooling, such as user feedback should be reported as `gen_ai.evaluation.user_feedback`.
+
+GenAI vendor-specific evaluation events SHOULD follow `gen_ai.{gen_ai.system}.evaluation.{evaluation method}` pattern.
+
+## User feedback evaluation
+
+The user feedback evaluation event SHOULD be captured if and only if user provided a reaction to GenAI model response.
+It SHOULD, when possible, be parented to the GenAI span describing such response.
+
+<!-- semconv gen_ai.evaluation.user_feedback -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+The event name MUST be `gen_ai.evaluation.user_feedback`.
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
```

Comment:
this probably should be in the common section and we should talk about user_feedback as an example.